backend/grants/admin.py

```diff
@@ -13,7 +13,6 @@
     "name",
     "full_name",
     "email",
-    "age",
```

Comment:
did you forget to add age_group here?

Reply:
No, I will look at the export in another PR, as I need to remove some stuff now that we have the user in the model.